examples/eks-msk-opensearch/terraform/templates/fluent-bit.yaml

```diff
@@ -0,0 +1,132 @@
+# Template out the image version and service account name to use so that they can be modified from Terraform.
+image:
+  tag: ${image_version}
+
+serviceAccount:
+  name: ${service_account_name}
+
+# Tolerate all taints so that all nodes are logged.
+tolerations:
+  - key: ""
+    operator: "Exists"
+    effect: "NoExecute"
+
+# Set hostNetwork to true and dnsPolicy to ClusterFirstWithHostNet so that fluent bit DaemonSet could call Kubelet locally.
+hostNetwork: true
+dnsPolicy: ClusterFirstWithHostNet
+
+# Node access required when using Kubelet
+rbac:
+  create: true
+  nodeAccess: true
+
+# Pipeline configuration.
+config:
+  # This is a config for the FluentBit application running the pipeline.
+  # We set the flush interval, log level and where our parsers are stored.
+  # We additionally expose the HTTP service so that Terraform can ping it.
+  service: |
+      [SERVICE]
+        Flush                     30
```

Comment:
Sorry i didn't notice this before. Why is the Flush interval so high? This will not be ideal for many high throughput envs. I'd like to keep the `Flush` as default. 

Also, you need to add `Grace 30` here since the default k8s SIGTERM to SIGKILL timeout is 30.